fetch-quotes.test.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable indent */
+import fetchQuotes from './fetch-quotes';
+
+describe('fetchQuotes', () => {
+
+    it('fetches a single quote from the futurama api', () => {
+
+    
+        const actual = fetchQuotes();
+    
+        expect(actual).toEqual(expect.any(Object));
```

Comment:
This `expect` statement is ultimately going to be way too general to be a useful test for this functionality - by using `expect.any(Object)` you end up saying a lot of things that are not what you want are acceptable results (for example, since an array is technically an object under the hood, a giant array of quotes will pass just the same as a single quote object).

The way to use expect.any here is to test for the shape of the received object:
```js
expect(actual).toEqual({
            name: expect.any(String),
            text: expect.any(String),
            image: expect.any(String)
        });
```